plutus-core/cost-model/data/models.R

```diff
@@ -160,14 +160,7 @@ modelFun <- function(path) {
         adjustModel (m, "AddInteger")
     }
 
-    subtractIntegerModel <- {
-        filtered <- data %>%
-            filter(BuiltinName == "SubtractInteger")  %>%
-            discard.upper.outliers("SubtractInteger") %>%
-            discard.overhead (two.args.overhead)
-        m <- lm(Mean ~ pmax(x_mem , y_mem), filtered)
-        adjustModel (m, "SubtractInteger")
-    }
+    subtractIntegerModel <- addIntegerModel
```

Comment:
shouldn't there be changes like this for the other ones where you're reusing models?

Reply:
No.  We were previously generating the data but then ignoring it and re-using other cost models, so there are already lines like this for those.